Content.Server/IoC/ServerContentIoC.cs

```diff
@@ -38,6 +38,7 @@
 using Content.Shared.Players.PlayTimeTracking;
 using Content.Shared.Players.RateLimiting;
 using Content.Shared.Starlight;
+using Content.Server.Economy;
```

Comment:
```suggestion
using Content.Server.Economy; // Starlight-edit
```

Pls add comments(Yeah, i know that we don't add it before, but anyway we must do that, why we don't do this before here? Idk)